src/main/resources/application.yml

```diff
@@ -4,7 +4,7 @@ spring.profiles.active: local
 spring:
   jpa:
     database: postgresql
-    database-platform: org.hibernate.spatial.dialect.postgis.PostgisPG95Dialect
+    database-platform: org.hibernate.dialect.PostgreSQLDialect
```

Comment:
> WARN 42661 --- [run-us] [           main] org.hibernate.orm.deprecation            : HHH90000025: PostgreSQLDialect does not need to be specified explicitly using 'hibernate.dialect' (remove the property setting and it will be selected by default)

로그보니 6번 database: postgresql, 7번 database-platform 설정 라인 둘다 삭제시키는게 좋을 것 같아요

6, 7번 라인 없으니 warning이 안 나오네요

Reply:
6,7번 라인이 없어서 추가했었는데 지금은 또 안 나네요😲